### ISSUE TITLE
.jsoneditor .jsoneditor-tree background is transparent

### ISSUE THREAD

**Issue by ChrisAcrobat** [josdejong/jsoneditor#1356]
Style of `.ace_content` defaults to white `#fff`, but none it set for `.jsoneditor-tree`. This make `.jsoneditor-tree` hard to read when background is dark or black. I have solved it locally, so I just letting you know. I tried to take a look, but I not that familiar with SCSS just yet to make a PR.

**Current values in browser**
```CSS
.jsoneditor .ace_content {
	background-color: #fff;
}
```
```CSS
.jsoneditor .jsoneditor-tree {
	background-color: initialt; /* not set/no value */
}
```

**Comment by ChrisAcrobat** [comment]
Somewhat related to #1295.

**Comment by josdejong** [comment]
Ah, you're right, we should give the `tree` mode a white background instead of transparent. I'll fix that.

**Comment by josdejong** [comment]
The fix is now published in v9.5.3
